src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -19736,7 +19736,76 @@ public class XAttribute : System.Attribute
                 );
         }
 
-        [Fact]
+        [ConditionalFact(typeof(CoreClrOnly))]
```

Comment:
I believe what you wanted to do was limited where the test binaries are actually executed, not where the test itself is run. I believe @cston has a bug somewhere on fixing all of the collection expression tests to follow that pattern.

Reply:
The tests in `CollectionExpressionTests` use an `IncludeExpectedOutput()` helper that allows executing test binaries on CoreClr only.

Reply:
Thanks, I think I've fixed the new tests.